go/worker/common/p2p/dispatch.go

```diff
@@ -62,11 +68,14 @@ type topicHandler struct {
 	p2p *P2P
 
 	topic       *pubsub.Topic
+	host        core.Host
 	cancelRelay pubsub.RelayCancelFunc
 	handlers    []Handler
 
 	numWorkers uint64
 
+	rawMsgQueue chan *rawMessage
```

Comment:
I think a more descriptive name like: `pendingQueue` (or something similar) would be better, as the type already tells that this is for `rawMsgs`.